android/src/paper/java/com/facebook/react/viewmanagers/RNGestureHandlerButtonManagerDelegate.java

```diff
@@ -23,7 +23,7 @@ public RNGestureHandlerButtonManagerDelegate(U viewManager) {
   public void setProperty(T view, String propName, @Nullable Object value) {
     switch (propName) {
       case "exclusive":
-        mViewManager.setExclusive(view, value == null ? false : (boolean) value);
+        mViewManager.setExclusive(view, value == null ? true : (boolean) value);
```

Comment:
They are flipped because the default value is supposed to be [`true`](https://github.com/software-mansion/react-native-gesture-handler/blob/80cb2a1ac5905218c69a69c227b205caa540d7b1/src/fabric/RNGestureHandlerButtonNativeComponent.ts#L19), and yes those are unrelated to this PR, should I extract them to a separate PR?